src/v0/destinations/snapchat_conversion/utilsV3.js

```diff
@@ -128,8 +128,7 @@ const getExtInfo = (message) => {
     deviceInfo.freeStorage,
     environmentInfo.timezone,
   ];
-
-  return extInfo.some((value) => value == null) ? null : extInfo;
+  return extInfo.map((value) => (value == null ? '' : value));
```

Comment:
I think the expectation will be to not send a value when it's not present.
So you also need to use filter for dropping the fields that are null. There's difference between the field being empty vs null vs 'null' vs non-existent field

Reply:
https://developers.snap.com/api/marketing-api/Conversions-API/Parameters As per the doc, I don't see sending an empty string, but the customer suggested sending empty strings.

Reply:
Extinfo requires all values and needs them to be in the order indexed below in the extinfo object specs.
It's assigning specific field [based on index](https://developers.snap.com/api/marketing-api/Conversions-API/Parameters#extinfo) - that's the reason for customer asking to send empty values to maintain the order. Then this change looks good